ci-operator/jobs/infra-periodics.yaml

```diff
@@ -643,6 +643,39 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-merge-robot
+- agent: kubernetes
+  cluster: api.ci
+  cron: '@daily'
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    app: image-pruner
+    ci.openshift.io/role: infra
+  name: periodic-ci-image-pruner
+  spec:
+    containers:
+    - args:
+      - adm
+      - prune
+      - images
+      - --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+      - --keep-tag-revisions=3
+      - --keep-younger-than=96h
+      - --confirm=true
+      command:
+      - oc
+      image: registry.svc.ci.openshift.org/ocp/4.7:cli
```

Comment:
This should be appci regisrty :)

Reply:
This job will be always on `api.ci`.
If the registry on `api.ci` is offline, then we do not need the job either.

Regardless, let us say, we want to use `registry.ci.openshift.org/ocp/4.7:cli` because it is preferred.
I think we need to link the secret with the auth for `registry.ci.openshift.org` to the service account `image-pruner` so the kubelet can download the image for the pod.
This whole thing is much easier on OCP4 where we have a global secret to config.